rainbow-me/rainbow
# Update balances onchain after tx confirm

Pulled out `fetchAssetsBalancesAndPrices` into its own action with optional pricing info from coingecko and polling.

That way the fallbackExplorer can continue using it in the same way, but we now have the option to trigger a one time only onchain balance update  on demand.

Added the logic to do so at the App.js level after we listen to a tx confirmation event.

PoW:


QA: 
- Affected balance should be updated around 5 seconds after the transaction was confirmed in the app.
- Send any asset to the selected account (from outside the app) and pull to refresh any time after the tx was confirmed. The balance should be updated

PoW: https://recordit.co/JMHQIoVKBg
